### PR TITLE
Fix gasnet-related test failures

### PR DESCRIPTION
**Pull request by Pansysk75**
Tests were broken (not running) due to an environment variable being set incorrectly.
@ct-clmsn Would this be ok?

### CODE REVIEW

**Comment by Pansysk75** [comment]
retest lsu

**Comment by ct-clmsn** [comment]
@Pansysk75 LGTM

**Comment by hkaiser** [comment]
@Pansysk75 Something is wrong here, see the failing tests, please.

File: cmake/HPX_AddTest.cmake [hunk 1]
```diff
@@ -179,7 +173,10 @@ function(add_hpx_test category name)
         add_test(NAME "${_full_name}" COMMAND ${cmd} "-p" "lci" "-r" "mpi"
                                               ${args}
         )
-        set_tests_properties("${_full_name}" PROPERTIES RUN_SERIAL TRUE)
+        set_tests_properties(
+          "${_full_name}" PROPERTIES RUN_SERIAL TRUE ENVIRONMENT
+                                     "GASNET_PSHM_NODES=2"
+        )
```
Comment (hkaiser) [inline cmake/HPX_AddTest.cmake:179]: Should this environment variable added only if running using the gasnet parcelport?

**Comment by hkaiser** [comment]
@Pansysk75 what's the status with this PR?

**Comment by Pansysk75** [comment]
> @Pansysk75 what's the status with this PR?

Still in progress (no particular blockers), I'll have a look as soon as I get the opportunity.

**Comment by Pansysk75** [comment]
@hkaiser I think we should merge this so that other PRs get tested properly (Assuming I didn't mess up anything else). Since gasnet might need some more work, I removed it from the testing for now, so that the tests give somewhat meaningful results.
We can tackle the remaining gasnet issues in a new PR.
Thanks a lot to @ct-clmsn for the help with this one

**Pull request merged**